Test failure in CI build 3207

The following test appears to have failed:

[#3207](https://circleci.com/gh/cockroachdb/cockroach/3207):

```
I0604 15:56:28.474259     313 node.go:264] Started node with [[]=] engine(s) and attributes []
I0604 15:56:28.474672     313 server.go:178] starting https server at 127.0.0.1:41243
I0604 15:56:28.475400     313 testserver.go:43] Test server listening on https: 127.0.0.1:41243
W0604 15:56:29.033404     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
I0604 15:56:29.036622     313 client.go:150] client 127.0.0.1:41243 connected
--- FAIL: TestSplitByMeta2KeyMax (1.40s)
	dist_sender_server_test.go:219: range split on Meta2KeyMax timed out
=== RUN TestMethods
W0604 15:56:29.194272     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
W0604 15:56:29.376281     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
W0604 15:56:29.659180     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
I0604 15:56:29.729504     313 rocksdb.go:88] opening in-memory rocksdb instance
I0604 15:56:29.735744     313 range_command.go:771] range 1: new leader lease replica 1:1 00:00:00.000 +1433433390.732s
I0604 15:56:29.743102     313 node.go:299] initialized store store=1:1 ([]=): {Capacity:257668677632 Available:199626338304 RangeCount:0 XXX_unrecognized:[]}
I0604 15:56:29.743264     313 node.go:229] new node allocated ID 1
I0604 15:56:29.743581     313 gossip.go:165] gossiping node descriptor node_id:"\001" address:<network:"tcp" address:"127.0.0.1:43605" > attrs:<> 
--
I0604 15:57:15.561490     313 node.go:264] Started node with [[]=] engine(s) and attributes []
I0604 15:57:15.561895     313 server.go:178] starting https server at 127.0.0.1:56820
I0604 15:57:15.562371     313 testserver.go:43] Test server listening on https: 127.0.0.1:56820
I0604 15:57:16.049625     313 client.go:150] client 127.0.0.1:56820 connected
--- PASS: TestKeysAndBodyArePreserved (1.49s)
FAIL
FAIL	github.com/cockroachdb/cockroach/kv	90.837s
=== RUN TestHeartbeatSingleGroup
--- PASS: TestHeartbeatSingleGroup (0.44s)
=== RUN TestHeartbeatMultipleGroupsJointLeader
--- PASS: TestHeartbeatMultipleGroupsJointLeader (0.04s)
=== RUN TestHeartbeatResponseFanout
I0604 15:55:59.054072     323 heartbeat_test.go:420] SubmitCommand succeed after Heartbeat Response fanout
E0604 15:55:59.055709     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
E0604 15:55:59.056019     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
E0604 15:55:59.056324     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
E0604 15:55:59.056583     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
I0604 15:56:28.474259     313 node.go:264] Started node with [[]=] engine(s) and attributes []
I0604 15:56:28.474672     313 server.go:178] starting https server at 127.0.0.1:41243
I0604 15:56:28.475400     313 testserver.go:43] Test server listening on https: 127.0.0.1:41243
W0604 15:56:29.033404     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
I0604 15:56:29.036622     313 client.go:150] client 127.0.0.1:41243 connected
--- FAIL: TestSplitByMeta2KeyMax (1.40s)
	dist_sender_server_test.go:219: range split on Meta2KeyMax timed out
=== RUN TestMethods
W0604 15:56:29.194272     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
W0604 15:56:29.376281     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
W0604 15:56:29.659180     313 http_sender.go:105] failed to send HTTP request with status code 503, 503 Service Unavailable
I0604 15:56:29.729504     313 rocksdb.go:88] opening in-memory rocksdb instance
I0604 15:56:29.735744     313 range_command.go:771] range 1: new leader lease replica 1:1 00:00:00.000 +1433433390.732s
I0604 15:56:29.743102     313 node.go:299] initialized store store=1:1 ([]=): {Capacity:257668677632 Available:199626338304 RangeCount:0 XXX_unrecognized:[]}
I0604 15:56:29.743264     313 node.go:229] new node allocated ID 1
I0604 15:56:29.743581     313 gossip.go:165] gossiping node descriptor node_id:"\001" address:<network:"tcp" address:"127.0.0.1:43605" > attrs:<> 
--
I0604 15:57:15.561490     313 node.go:264] Started node with [[]=] engine(s) and attributes []
I0604 15:57:15.561895     313 server.go:178] starting https server at 127.0.0.1:56820
I0604 15:57:15.562371     313 testserver.go:43] Test server listening on https: 127.0.0.1:56820
I0604 15:57:16.049625     313 client.go:150] client 127.0.0.1:56820 connected
--- PASS: TestKeysAndBodyArePreserved (1.49s)
FAIL
FAIL	github.com/cockroachdb/cockroach/kv	90.837s
=== RUN TestHeartbeatSingleGroup
--- PASS: TestHeartbeatSingleGroup (0.44s)
=== RUN TestHeartbeatMultipleGroupsJointLeader
--- PASS: TestHeartbeatMultipleGroupsJointLeader (0.04s)
=== RUN TestHeartbeatResponseFanout
I0604 15:55:59.054072     323 heartbeat_test.go:420] SubmitCommand succeed after Heartbeat Response fanout
E0604 15:55:59.055709     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
E0604 15:55:59.056019     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
E0604 15:55:59.056324     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection
E0604 15:55:59.056583     323 transport.go:202] sending rpc failed: read tcp 127.0.0.1:55303: use of closed network connection

```
Please assign, take a look and update the issue accordingly.

--- a/kv/dist_sender_server_test.go
+++ b/kv/dist_sender_server_test.go
@@ -215,7 +215,7 @@ func TestSplitByMeta2KeyMax(t *testing.T) {
 
 	select {
 	case <-ch:
-	case <-time.After(500 * time.Millisecond):
+	case <-time.After(5 * time.Second):
 		t.Error("range split on Meta2KeyMax timed out")
 	}
 }

